pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -233,6 +233,9 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeMonopod(f, monopod.Pod)...)
 		}
 
+		bcPipelines, _ := graphview.AllImagePipelinesFromBuildConfig(g, graphview.IntSet{})
```

Comment:
Isn't this redundant information from what we are already showing?   I like the general idea, but I don't like displaying redundant info.  Is a build chain useful to me?  What about all the other inputs to the build chain?  What do we do when there are multiple inputs to the same build (because of image sources)?